src/model/trust.model.ts

```diff
@@ -31,10 +40,11 @@ export interface Trust {
   INDIVIDUALS?: TrustIndividual[];
   HISTORICAL_BO?: TrustHistoricalBeneficialOwner[];
   CORPORATES?: TrustCorporate[];
+  TRUSTEES?: (GeneralTrustee | InterestedPersonTrustee)[];
```

Comment:
This is unrequired. The individual trustees from the trust individual beneficial owner page are stored in `INDIVIDUALS?: TrustIndividual[];`.